pkg/kubelet/status/status_manager.go

```diff
@@ -567,9 +567,9 @@ func (m *manager) syncPod(uid types.UID, status versionedPodStatus) {
 		return
 	}
 	if unchanged {
-		klog.V(3).Infof("Status for pod %q is up-to-date: (%d)", format.Pod(pod), status.version)
+		klog.V(3).Infof("Status for pod %q is up-to-date: (version: %d)", format.Pod(pod), status.version)
 	} else {
-		klog.V(3).Infof("Status for pod %q updated successfully: (%d, %+v)", format.Pod(pod), status.version, status.status)
+		klog.V(3).Infof("Status for pod %q updated successfully: (version: %d, phase: %s)", format.Pod(pod), status.version, status.status.Phase)
```

Comment:
this may not be ideal to troubleshoot the pod readiness ++ feature where pod condition is set from the API server and than used to evaluate pod readiness. https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-readiness-gate

As you mentioned in description, adding a formatter may be a good way to introduce some logic on what is important in podPhase and what is less important